CONTRIBUTING.md

```diff
@@ -0,0 +1,77 @@
+[![License MIT](https://img.shields.io/badge/license-MIT-brightgreen.svg)](https://opensource.org/licenses/MIT)
+[![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/org-formation/org-formation-cli/issues)
+
+## How to contribute to Org-formation
+
+### **Did you find a bug?**
+
+* **Ensure the bug was not already reported** by searching on GitHub under [Issues](https://github.com/org-formatio/org-formation-cli/issues).
+
+* If you're unable to find an open issue addressing the problem, [open a new one](https://github.com/org-formatio/org-formation-cli/issues/new).
+  Be sure to include a **title and clear description**, as much relevant information as possible, and a **project sample**
+  demonstrating the expected behavior that is not occurring.
+  
+### **Did you write a patch that fixes a bug?**
+
+* Open a new GitHub pull request with the patch.
+
+* Ensure the PR description clearly describes the problem and solution. Include the relevant issue number if applicable.
+
+### **Do you intend to add a new feature or change an existing one?**
+
+* Suggest your change in the [org-formation slack channel](https://org-formation.slack.com).
```

Comment:
will this link work for people that dont have the invite link? if not maybe use this link instead (https://join.slack.com/t/org-formation/shared_invite/enQtOTA5NjM3Mzc4ODUwLTMxZjYxYzljZTE5YWUzODE2MTNmYjM5NTY5Nzc3MzljNjVlZGQ1ODEzZDgyMWVkMDg3Mzk1ZjQ1ZjM4MDhlOGM)

Reply:
I have added "Need to ask questions?" section with this join link